ouroboros-consensus/src/Ouroboros/Storage/ChainDB/Impl/VolDB.hs

```diff
@@ -494,22 +514,36 @@ blockFileParser' :: forall m blk h. (IOLike m, HasHeader blk)
                  -> (blk -> IsEBB)
                  -> (blk -> BinaryInfo Encoding)
                  -> (forall s. Decoder s (Lazy.ByteString -> blk))
+                 -> (blk -> Bool)
                  -> VolDB.Parser
-                     Util.CBOR.ReadIncrementalErr
+                     (BlockFileParserError (HeaderHash blk))
                      m
                      (HeaderHash blk)
-blockFileParser' hasFS isEBB encodeBlock decodeBlock = VolDB.Parser $
-       fmap (fmap (fmap fst)) -- Drop the offset of the error
-     . Util.CBOR.readIncrementalOffsets hasFS decoder'
```

Comment:
Yeah, if `readIncrementalOffsets` is no longer used, I would remove it. We can always resurrect it if necessary.